### PR TITLE
Set05| Add Node Documentation for...

### PR DESCRIPTION
**Pull request by johnpierson**
### Purpose

Add node documentation for:

Material.ByName
Material.Color
Material.CutPatternColor
Material.CutPatternId
Material.MaterialClass
Material.Name
Material.Shininess
Material.Smoothness
Material.SurfacePatternColor
Material.Transparency

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [x] Snapshot of UI changes, if any.

### Reviewers
@achintyabhat 

### FYIs
@Amoursol

### CODE REVIEW

**Review by Mikhinja — APPROVED** [review]

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.Material.Color.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## In Depth
+`Material.Color` returns a Dynamo color that represents the material's cut pattern color (if available).
+
```
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.Material.Color.md:3]: What is the difference between Material.color node and Material.CutPatternColor node? They both seem to do the same thing.

**Review by achintyabhat — CHANGES_REQUESTED** [review]
This set is almost good to go. Just wasn't sure about one of the nodes. Can you please confirm if the description is accurate? Seems like there are two nodes which perform the same thing.

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.Material.Color.md (hunk 1 above)
Comment (johnpierson) [reply doc/distrib/NodeHelpFiles/_review/Revit.Elements.Material.Color.md:3]: Added some additional text to distinguish as I believe material.color returns the default shading color

**Comment by johnpierson** [comment]
Fixed!

**Comment by johnpierson** [comment]
````Material.Color` returns a Dynamo color that represents the material's shading color. If the material's shading color is set to "use render appearance", that value is returned.```

**Review by achintyabhat — APPROVED** [review]

**Pull request merged**